analyticsdataserver/tests/test_clients.py

```diff
@@ -1,28 +1,39 @@
-import json
 import unittest.mock as mock
+from urllib.parse import urljoin
 
 import responses
+from django.conf import settings
 from django.test import TestCase
 from requests.exceptions import ConnectionError as RequestsConnectionError
 
 from analyticsdataserver.clients import CourseBlocksApiClient
 
 
 class ClientTests(TestCase):
-    @mock.patch('analyticsdataserver.clients.EdxRestApiClient')
-    def setUp(self, restApiClientMock):
-        self.client = CourseBlocksApiClient('http://example.com/', 'token', 5)
+
+    def setUp(self):
+        self.client = CourseBlocksApiClient(settings.LMS_BASE_URL, 'client_id', 'client_secret')
+        responses.add(
+            responses.POST,
+            urljoin(settings.LMS_BASE_URL, 'oauth2/access_token'),
```

Comment:
Should we add a trailing back slash to the base_url param in urljoin function?

Reply:
it's LMS_BASE_URL, so with or without adding a trailing slash, urljoin will work the same in both cases